source/03-components/accordion/_accordion.scss

```diff
@@ -0,0 +1,2 @@
+// @file
+// Component: Accordion Group
```

Comment:
Minor nitpick: I think “Accordion” is a better name.

Reply:
Changed